Sources/XCBuildSupport/PIFBuilder.swift

```diff
@@ -611,6 +611,9 @@ final class PackagePIFProjectBuilder: PIFProjectBuilder {
 
             // Also propagate this search path to all direct and indirect clients.
             impartedSettings[.HEADER_SEARCH_PATHS, default: ["$(inherited)"]].append(clangTarget.includeDir.pathString)
+            
+            let publicHeaders = clangTarget.headers.filter { $0.isDescendant(of: clangTarget.includeDir) }
```

Comment:
Is this the correct way to collect public headers?

Reply:
I *think* this looks right, but I want to double-check